app/DoctrineMigrations/Version20160313161454.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160313161454 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE fos_user CHANGE want_to_learn want_to_learn TINYINT(1) DEFAULT NULL');
```

Comment:
No, it is a boolean value. The best solution is to move those settings to a user settings table. But in this case I think it's easier to set the data option. 
```php
'data' => $this->securityContext->hasRole('ROLE_USER_COMPLETED') ? $builder->getData()->getWantToLearn() : null
```
Not sure about the exact syntax but something like that. You have to inject the securityContext service by configuring the form type as a service.